File: src/p1013/java/solution.java
class Solution {
    public int numPairsDivisibleBy60(int[] times) {
        int result = 0, counts[] = new int[60];
        for (int time : times) {
            time %= 60;
            result += counts[(60 - time) % 60];
            counts[time]++;
        }
        return result;
    }
}

File: src/p1013/python/solution.py
from typing import List


class Solution:
    def numPairsDivisibleBy60(self, times: List[int]) -> int:
        result, counts = 0, [0] * 60
        for time in times:
            time %= 60
            result += counts[(60 - time) % 60]
            counts[time] += 1
        return result
